packages/sdk/src/Authentication.ts

```diff
@@ -40,8 +39,8 @@ export const createAuthentication = (config: Pick<StrictStreamrClientConfig, 'au
         const provider = new BrowserProvider(ethereum)
         const signer = provider.getSigner()
         return {
-            getAddress: pMemoize(async () => {
-                return toEthereumAddress(await (await signer).getAddress())
+            getUserId: pMemoize(async () => {
```

Comment:
why not leave the getAddress though? It could return same as getUserId, or `toEthereumAddress` or something, maybe throwing if the user id is NOT an Ethereum address.